src/Microsoft.AspNetCore.Authentication/TokenExtensions.cs

```diff
@@ -63,6 +63,26 @@ public static string GetTokenValue(this AuthenticationProperties properties, str
                 : null;
         }
 
+        public static bool UpdateTokenValue(this AuthenticationProperties properties, string tokenName, string tokenValue)
```

Comment:
Should there be a way to add a new token as well? and/or set a token regardless of if it already existed?